components/Navbar.vue

```diff
@@ -255,7 +254,6 @@ const openWalletConnectModal = (): void => {
 }
 
 const showMobileNavbar = () => {
-  document.body.classList.toggle('is-clipped')
```

Comment:
There is a reason why I added this line but I can't remember why (ref #6382)

Reply:
Ok now I remember, on Mobile if you remove this line, you can still scroll through explore page even if sidebar is open & card menu header is looking bad

![Screenshot 2023-08-11 at 10-37-27 Low minting fees and carbonless NFTs KodaDot - NFT Market Explorer](https://github.com/kodadot/nft-gallery/assets/9987732/cdcd7244-c793-4929-996c-6d2345a7e88b)

Reply:
oh ok @roiLeo , will check it out further

Reply:
@roiLeo I can confirm that without the is-clipped toggle, you can still scroll through explore page even if the sidebar menu is opened. 

If we want the is-clipped toggle then, I can save the scroll location before the sidebar opens and restore it when sidebar closes. Is this okay?

Reply:
the lastScrollPosition is already saved here so i just restored the position everytime the sidebar closes.

Reply:
Still same issue for me

Reply:
Here's a recording of mine with the latest deployment on netlify

https://deploy-preview-6673--koda-canary.netlify.app/

https://github.com/kodadot/nft-gallery/assets/3810177/c2796b6d-fadc-4b1c-9212-265ae2d37e02

Might be a browser issue?
What browser are you using?

Reply:
<img width="1116" alt="image" src="https://github.com/kodadot/nft-gallery/assets/3810177/6748822f-3e02-492b-ba1b-3d24a60c99c6">

Reply:
code is very straight forward
document.documentElement.scrollTop = lastScrollPosition.value

It should be supported by almost every browsers

Reply:
cleared cache now it work